roboteam_ai/src/control/ControlUtils.cpp

```diff
@@ -262,6 +271,16 @@ Vector2 ControlUtils::calculateForce(rtt::Vector2 vector, double weight, double
     return {0, 0};
 }
 
+/// Calculates the position the robot should "aim" at given the ballPos, a targetPos and the geneva state
+Vector2 ControlUtils::getGenevaAim(Vector2 ballPos, Vector2 targetPos, int genevaState) {
+    double angleToTarget = (targetPos - ballPos).angle();
+    double genevaAngle = genevaMap[genevaState];
+    double aimAngle = constrainAngle(angleToTarget - genevaAngle);
```

Comment:
why -geneva Angle and not + genevaAngle? The aim should be at the addition, I think right?

Reply:
perhaps you should invert the geneva map to not have the geneva rotation itself but rather the direction it aims at as the output (so instead of 1-5 corresponding to -20 -> 20 let 1-5 go to 20->-20